src/validation-group.js

```diff
@@ -408,16 +408,16 @@ export class ValidationGroup {
    * @param threshold {Object} An optional threshold that will be passed to the customFunction
    * @returns {ValidationGroup} returns this ValidationGroup, to enable fluent API
    */
-  passes(customFunction, threshold) {
+  passes(customFunction: Function, threshold: any): ValidationGroup {
```

Comment:
instead of `Function` correct type (at least based on documentation) would be `(newValue, threshold) => boolean`. 
If You have time to look up types of newValue and threshold, then You could add them as well